operations/admin.go

```diff
@@ -476,3 +481,70 @@ func updateRoleCmd() cli.Command {
 		},
 	}
 }
+
+func adminBackup() cli.Command {
+	const (
+		collectionNameFlag = "collection"
+		prefixFlagName     = "prefix"
+	)
+
+	return cli.Command{
+		Name:  "backup",
+		Usage: "create a backup (to s3) of a collection",
+		Flags: mergeFlagSlices(
+			serviceConfigFlags(),
+			addDbSettingsFlags(
+				cli.StringFlag{
+					Name:  joinFlagNames(collectionNameFlag, "-c"),
+					Usage: "specify the name of the collection to backup",
+				},
+				cli.StringFlag{
+					Name:  "prefix",
+					Value: "archive",
+					Usage: "specify prefix within the bucket to store this archive",
+				},
+			),
+		),
+		Before: requireStringFlag(collectionNameFlag),
+		Action: func(c *cli.Context) error {
+			ctx, cancel := context.WithCancel(context.Background())
+			defer cancel()
+
+			collectionName := c.String(collectionNameFlag)
+
+			env, err := evergreen.NewEnvironment(ctx, c.String(confFlagName), parseDB(c))
```

Comment:
Isn't this the wrong conf to construct an environment from? This is the user's local conf. Is your plan to provide one on an app server for the db values?

Reply:
nope. that's c.Parent().String(confFlagName)

Reply:
this one comes from the serviceConfigFlags